components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -2333,18 +2337,30 @@ subroutine ice_cldliq_wet_growth(rho,t,pres,rhofaci,    &
 
    real(rtype) :: qsat0, dum, dum1
 
+#ifdef SCREAM_CONFIG_IS_CMAKE
+    if (use_cxx) then
+       call ice_cldliq_wet_growth_f(rho,t,pres,rhofaci, &
+                                  f1pr05,f1pr14,xxlv,xlf,dv,kap,mu,sc, &
+                                  qv,qc_incld,qitot_incld,nitot_incld,qr_incld, &
+                                  log_wetgrowth,qrcol,qccol,qwgrth,nrshdr,qcshd)
+       return
+    endif
+#endif
 
    if (qitot_incld.ge.qsmall .and. qc_incld+qr_incld.ge.1.e-6_rtype .and. t.lt.zerodegc) then
       qsat0  = 0.622_rtype*e0/(pres-e0)
 
-      qwgrth = ((f1pr05 + f1pr14*sc**thrd*(rhofaci*rho/mu)**0.5_rtype)*       &
+      qwgrth = ((f1pr05 + f1pr14*bfb_cbrt(sc)*bfb_pow((rhofaci*rho/mu),0.5_rtype))*       &
```

Comment:
Here instead of bfb_pow(.,0.5) you should make a bfb_sqrt function, since we'll use std::sqrt in C++.